proto/regen/ecocredit/marketplace/v1/state.proto

```diff
@@ -20,7 +20,7 @@ message SellOrder {
   // id is the unique ID of sell order.
   uint64 id = 1;
 
-  // seller is the bytes address of the owner of the credits being sold.
+  // seller is the address of the account that is selling credits.
```

Comment:
```suggestion
  // seller is the address (in bytes) of the account that is selling credits.
```
maybe it still makes sense to explicitly state that its in byte representation?

Reply:
I don't think it's necessary. I previously made a change to add something like "unique string identifier" in relation to credit class IDs and it was pointed out that this is excessive because the field clearly states the type. I would have agreed before but I don't think we need to state the type unless we are adding details such as "any arbitrary string".